### DISCUSSION TITLE
Cannot submit reviews without approval or top level comment

### DISCUSSION THREAD

**Discussion by anuraaga** [community/community#114799] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Since the latest beta, there seems to be a new review experience. However, a common flow seems to not work. When I add line comments on the code and then try to submit the review, the submit button is disabled. It is enabled if I select approve (don't want to approve) or add a top level review comment, which normally I shouldn't need. Possibly related is once I edit the review comment, if I try to delete it doesn't work since the edit button is disabled when that field is empty.

![Screenshot_20240330-092818](https://github.com/community/community/assets/198344/643ffec5-ae19-4fe5-b27f-e8400b3c32f4)

**Comment by alcere** [comment, marked as answer]
Hi @anuraaga! Thank you so much for taking the time to provide us with your feedback on our beta feature! We truly appreciate it. We've identified the issue and we're working hard to include a fix in our upcoming beta release. Please, continue sharing your thoughts with us, as your feedback is incredibly valuable and helps us improve our product.
